scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/quartz/AppScalingScheduleJob.java

```diff
@@ -31,10 +30,7 @@ protected void executeInternal(JobExecutionContext context) throws JobExecutionE
 		Long scheduleId = dataMap.getLong("scheduleId");
 		Object schedule = dataMap.get("schedule");
 
-		logger.info("Job Key is " + jobKey);
-		logger.info("Application Id: " + appId + " Schedule Id: " + scheduleId);
-		logger.info("Schedule: " + schedule);
-		logger.info("==========================================");
-
+		logger.info(String.format("Job Key: %s, Application Id: %s, Schedule Id: %s, Schedule: %s", jobKey, appId,
```

Comment:
This should be combined with the above. Otherwise its very confusing to have 2 separate lines that can be mixed with other things.
Also, you should use logger.info(msg, o1, o2, o3, ..) no need to format it.